include/xsimd/types/xsimd_neon_double.hpp

```diff
@@ -580,6 +580,18 @@ namespace xsimd
                 return vbslq_f64(cond, a, b);
             }
 
+#if XSIMD_ARM_INSTR_SET >= XSIMD_ARM8_64_NEON_VERSION
+            static batch_type zip_lo(const batch_type& lhs, const batch_type& rhs)
+            {
+                return vzip1q_f64(lhs, rhs);
+            }
+
+            static batch_type zip_hi(const batch_type& lhs, const batch_type& rhs)
+            {
+                return vzip2q_f64(lhs, rhs);
+            }
+#endif
+
```

Comment:
I think we should provide a fallback implementation when `XSIMD_ARM_INSTR_SET < XSIMD_ARM8_64_VERSION`. Otherwise the code will fail to compile if the user calls the `zip_lo` / `zip_hi` methods

Reply:
Really thanks for your comments.
In my original thought, it's meaningless for users to call `zip_lo/hi` on Arm_32 which does not support ARM8_64.
And consider that: 
1. when user calls `zip_lo/hi ` on `x86`,  it's ok that Arm `zip_lo/hi` implementation would not be compiled.
2. when user calls `zip_lo/hi ` on `ARM8_64` , that's definitely ok.  

So I didn't add the fallback implementation of `zip_lo/hi` .

But for code consistency and unexpected user's calling on Arm_32, it makes sense to add fallback implementation to source code tree .
I'll add the fallback implementation soon.

On the other hand, it might suffer performance loss if user calls fallback implementation of `zip_lo/hi ` on < `ARM8_64`.
IMHO, we should explain it in our documentation to prevent users from calling `zip_lo/hi` on Arm_32. 

WDYT where we should put the explanation in ?  
Comments directly in the code or in [xsimd/docs/](https://github.com/xtensor-stack/xsimd/tree/master/docs)?